tsconfig.json

```diff
@@ -4,7 +4,6 @@
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "baseUrl": "src",
```

Comment:
This is the critical line--when `baseUrl` is specified, VSCode will suggest imports relative to this directory, which will break at runtime.